bin/impl/start-accumulo-hadoop-zookeeper.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+# Copyright 2014 Uno authors (see AUTHORS)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -z "$(pgrep -f QuorumPeerMain)" ]]; then
```

Comment:
I agree with Keith's comment.  Could rename this file to `start.sh` and it could accept argument so command is `uno start <component>`.  The stop command could also follow this format.  For fluo & metrics you could just print it's unsupported at this time and exit.

```bash
function start_accumulo() {
if [[ -z "$(pgrep -f QuorumPeerMain)" ]]; then
	"$ZOOKEEPER_HOME"/bin/zkServer.sh start
fi
}

function start_hadoop() ()
if [[ -z "$(pgrep -f hadoop\\.hdfs)" ]]; then
	"$HADOOP_PREFIX"/sbin/start-dfs.sh
fi
if [[ -z "$(pgrep -f hadoop\\.yarn)" ]]; then
	"$HADOOP_PREFIX"/sbin/start-yarn.sh
fi
}

function start_accumulo() {
if [[ -z "$(pgrep -f accumulo\\.start)" ]]; then
	if [[ $ACCUMULO_VERSION =~ ^1\..*$ ]]; then
	  "$ACCUMULO_HOME"/bin/start-all.sh
	else
	  "$ACCUMULO_HOME"/bin/accumulo-cluster start
	fi
fi
}

case "$1" in
accumulo)
   start_zookeeper
   start_hadoop
   start_accumulo
   ;;
zookeeper)
   start_zookeeper
   ;;
hadoop)
   start_hadoop
   ;;
fluo)
   echo "Fluo is not supported by 'start' command at this time"
   exit 1
   ;;
*)
   echo "Unknown component give to start command: $1"
   exit 1
   ;;
esac